hikari/impl/entity_factory.py

```diff
@@ -1896,6 +1896,7 @@ def deserialize_member_presence(  # noqa: CFQ001 - Max function length
                 secrets=secrets,
                 is_instance=activity_payload.get("instance"),  # TODO: can we safely default this to False?
                 flags=presence_models.ActivityFlag(activity_payload["flags"]) if "flags" in activity_payload else None,
+                buttons=activity_payload.get("buttons") or [],
```

Comment:
Could this be `activity_payload.get("buttons", [])`?

Reply:
that approach would always build an empty list regardless of whether buttons is present or not and then discard it if buttons is present